src/main/java/racingcar/entity/Car.java

```diff
@@ -0,0 +1,31 @@
+package racingcar.entity;
+
+import static racingcar.util.RandomUtil.checkMoveOrWait;
+
+public class Car implements Comparable<Car> {
+    private final String name;
+    private int position = 0;
+
+    public Car(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public int getPosition() {
+        return position;
+    }
+
+    public void progress(){
+        if(checkMoveOrWait()) {
```

Comment:
다른데는 다 띄우셨는데 안 띄우셔서 코멘트 남겨요~

```suggestion
        if (checkMoveOrWait()) {
```

Reply:
수정 했습니다!